sdk/purview/azure-analytics-purview-datamap/README.md

```diff
@@ -0,0 +1,65 @@
+# Azure PurviewDataMap client library for Java
+
+Azure PurviewDataMap client library for Java.
+
+This package contains Microsoft Azure PurviewDataMap client library.
```

Comment:
Please be more descriptive on what does purview datamap do and what does the library do.

Reply:
updated